worker/src/RTC/Transport.cpp

```diff
@@ -2306,56 +2306,49 @@ namespace RTC
 	{
 		MS_TRACE();
 
-		RTC::RTCP::CompoundPacket* packet{ nullptr };
+		RTC::RTCP::CompoundPacket::UniquePtr packet(nullptr);
 
 		for (auto& kv : this->mapConsumers)
 		{
 			auto* consumer = kv.second;
 
 			for (const auto& rtpStream : consumer->GetRtpStreams())
 			{
-				RTC::RTCP::CompoundPacket::ReturnIntoPool(packet);
-				packet = consumer->GetRtcp(rtpStream, nowMs);
+				auto packet = consumer->GetRtcp(rtpStream, nowMs);
```

Comment:
Shouldn't this be?:
```suggestion
				packet = consumer->GetRtcp(rtpStream, nowMs);
```
Or even this?:
```suggestion
				packet.reset(consumer->GetRtcp(rtpStream, nowMs));
```

Reply:
Thank you!